src/app/common/templates/config-data.yaml

```diff
@@ -36,6 +36,7 @@ CommandHandlerInterfaceOnlyClusters:
     - Microwave Oven Control
     - Energy EVSE
     - Energy EVSE Mode
+    - Meter Identification
```

Comment:
This is a claim about implementation.  There is no implementation in this PR, so this part should not be here.